contracts/name-minter/src/integration_tests.rs

```diff
@@ -1047,6 +1048,33 @@ mod collection {
             .unwrap();
         assert_eq!(res.extension.records[0].name, name.to_string());
         assert_eq!(res.extension.records[0].verified, Some(true));
+
+        // verify something as false
+        let msg = SgNameExecuteMsg::VerifyTextRecord {
```

Comment:
This is not part of the rate limiter test. Please move it into a test of its own.